_data/_events.json

```diff
@@ -0,0 +1,239 @@
+[
+   {
+      "name":"",
+      "id":"",
+      "title":"",
+      "description":"",
+      "callsForPapersDeadline":"",
+      "location":"",
+      "startDate":"",
+      "endDate":"",
+      "date":"",
+      "openingParagraph":"",
+      "pastOpeningParagraph":"",
+      "blockquotes":[
+         ""
+      ],
+      "urlsAfterBlockquote":[
+         ""
+      ],
+      "locationHeader":"",
+      "locationName":"",
+      "linkHeader":"",
+      "linkUrl":[
+         ""
+      ],
+      "impDatesHeader":"",
+      "impDatesHeaderH4":[
+         {
+            "h4":"",
+            "table":[
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               }
+            ]
+         },
+         {
+            "h4":"",
+            "table":[
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               }
+            ]
+         },
+         {
+            "h4":"",
+            "table":[
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               }
+            ]
+         },
+         {
+            "h4":"",
+            "table":[
+               {
+                  "name":"",
+                  "date":""
+               },
+               {
+                  "name":"",
+                  "date":""
+               }
+            ]
+         }
+      ],
+      "table":[
+         {
+            "name":"",
+            "date":""
+         },
+         {
+            "name":"",
+            "date":""
+         }
+      ],
+      "bulletKeyNoteSpeakOrTopsHeader":"",
```

Comment:
Again, we should have this.

Instead of ""bulletKeyNoteSpeakOrTopsHeader" and "names", we need just

```
"organisers": [
   {
       "name": "...",
       "institution": "..."
   },
   ...
]
```